js/src/components/app-text/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './index.scss';
+
+/**
+ * Renders a paragraph with some predefined styles based on the variant
```

Comment:
:nail_care: It's not necessarily a paragraph, right?
```suggestion
 * Renders a paragraph or given element with some predefined styles based on the variant
```